### PR TITLE
Support category selection redux setup

### PR DESCRIPTION
**Pull request by winterDroid**
## Summary

Support category selection redux setup

Part of #2362

### CODE REVIEW

File: assets/src/edit-story/app/media/media3p/categories/actions.js [hunk 1]
```diff
@@ -47,3 +47,17 @@ export const fetchCategoriesError = (dispatch) => ({ provider }) => {
     payload: { provider: provider },
   });
 };
+
+export const selectCategory = (dispatch) => ({ provider, category }) => {
```
Comment (diegovar) [inline assets/src/edit-story/app/media/media3p/categories/actions.js]: Should this be categoryName?

File: assets/src/edit-story/app/media/media3p/categories/reducer.js [hunk 2]
```diff
@@ -62,6 +63,19 @@ function reducer(state = INITIAL_STATE, { type, payload }) {
         isLoaded: true,
       };
     }
+    case types.SELECT_CATEGORY: {
+      const { category } = payload;
+      return {
+        ...state,
+        selectedCategory: category,
```
Comment (diegovar) [inline assets/src/edit-story/app/media/media3p/categories/reducer.js]: Same here, selectedCategoryName?

File: assets/src/edit-story/app/media/media3p/categories/actions.js (hunk 1 above)
Comment (winterDroid) [reply assets/src/edit-story/app/media/media3p/categories/actions.js]: Done.

File: assets/src/edit-story/app/media/media3p/categories/reducer.js (hunk 2 above)
Comment (winterDroid) [reply assets/src/edit-story/app/media/media3p/categories/reducer.js]: Done.

**Review by diegovar — APPROVED** [review]

**Pull request merged**